cloud/tests/conftest.py

```diff
@@ -0,0 +1,91 @@
+import boto3
+import os
+import pytest
+
+
+class CloudTestHelper:
+
+    @staticmethod
+    def prepare_data(*args):
+        s3 = boto3.client('s3')
+        for arg in args:
+            s3.download_file('autogluon-cloud', arg, os.path.basename(arg))
```

Comment:
if possible use keyword arguments here so it is easier to understand

Reply:
Here we basically get args as name of the file being downloaded from s3 bucket. I think it makes more sense to use *args directly as there's no variable name involved.

Reply:
I don't mean that, I mean for the `s3.download_file(...)` call

Reply:
regardless, this is a nit comment, so feel free to ignore